.pre-commit-config.yaml

```diff
@@ -52,6 +52,18 @@ repos:
       - id: python-check-blanket-noqa
         # Enforce that all noqa annotations always occur with specific codes.
 
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.2.0
+    hooks:
+      - id: pyupgrade
+        args: ["--py38-plus"]
```

Comment:
We may be bumping to py3.9+ soon.

Reply:
I just moved it to be before `isort` and `black` so that one should only have to run pre-commit once. In my experience its change sometimes get reformatted slightly by `black` in particular).